pkg/codegen/internal/test/testdata/external-resource-schema/dotnet/ArgFunction.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Immutable;
 using System.Threading.Tasks;
 using Pulumi.Serialization;
+using Pulumi.Example;
```

Comment:
Since everything is being emitted inside `namespace Pulumi.Example { ... }`, there shouldn't be a need for `using Pulumi.Example;`